### PR TITLE
Adds a check for java >=8 and tries curl for parser setup

### PR DESCRIPTION
**Pull request by netj**
This prevents HazyResearch/deepdive#505 and related issues.

### CODE REVIEW

File: parser/setup.sh [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
 
 # fetch SR models
-DESTDIR=$DIRNAME/lib
+DESTDIR="$DIRNAME"/lib
 FILENAME='stanford-srparser-2014-10-23-models.jar'
 if [ ! -e "$DESTDIR/$FILENAME" ]; then
-    mkdir -p $DESTDIR
-    wget -P $DESTDIR http://nlp.stanford.edu/software/stanford-srparser-2014-10-23-models.jar
+    mkdir -p "$DESTDIR"
+    url="http://nlp.stanford.edu/software/stanford-srparser-2014-10-23-models.jar"
```
Comment (alldefector) [inline parser/setup.sh]: Do we want to use the latest model?

**Comment by alldefector** [comment]
LGTM!

File: parser/setup.sh (hunk 1 above)
Comment (netj) [inline parser/setup.sh]: I think we can do that later when it becomes necessary?

**Comment by netj** [comment]
Thanks for reviewing!

**Pull request merged**